### PR TITLE
blog: August 2018 newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Better late than never! Feel free to fix typos etc on merge :)

@ehelms - anything else need to be said about the containers?
@sbernhard - need anything else on the FrosCon coverage?

### CODE REVIEW

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+to read here, so it's broken down into multiple posts. You can get the
+top-level over on [this forum
+post](https://community.theforeman.org/t/containerizing-the-foreman-ecosystem/10948)
+and then dive into details from there.
+
+I'd recommend for *anyone* with an interest in the future direction of Foreman
+to read over these, and give their thoughts on any part of it which interests
+you. Thanks!
+
+### Foreman at FrosCon
```
Comment (sbernhard) [inline _posts/2018-08-31-foreman-community-newsletter---august-2018.md]: The name of the conference is "FrOSCon".

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+Thanks to the awesome work of Matthias, Bernhard, and the rest of the ATIX
+team, Foreman had a community presence at FrosCon in Bonn this year. Whilst I
```
Comment (sbernhard) [inline _posts/2018-08-31-foreman-community-newsletter---august-2018.md]: FrOSCon

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+### Releases for everyone!
+
+We've had a lot of releases this month, so I'll group them all together. We've released:
+
+* [Foreman 1.19.0][1_19_0] was released on the very last day of August,
+  featuring support for Ubuntu Bionic, syslog logging, reporting failed builds,
+  and more ### Foreman 1.18.0 released
+* [1.17.3 and 1.17.4][1_17_4] bug fix releases. If you are still on 1.17, be
+  sure to upgrade to 1.17.4 before upgrading to 1.18!
+* [Katello 3.8 RC 2 & 3][k_3_8] - do give the RC a try and let us know how you find it :)
```
Comment (ekohl) [inline _posts/2018-08-31-foreman-community-newsletter---august-2018.md:25]: https://github.com/theforeman/foreman-packaging/commit/473f1d9c4030112be8420f224d54359a83a9c9e0 was merged. Not sure if it's been officially released though.

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+As ever, many thanks to the contributors to these releases - coders, RC testers,
+documentation writers, and bug/feature reporters. Foreman wouldn't be what it
+is without you! If you do find issues with, please do [report them!][issues]
+
+Big thanks also to Ondrej, Ewoud, and Tomer for their effort in bringing these
+releases out at this rate, there's been a *lot* of work behind the scenes.
+
+### New RFCs for container work
+
+In the [survey analysis]() I posted earlier this year, I called attention to
```
Comment (ekohl) [inline _posts/2018-08-31-foreman-community-newsletter---august-2018.md]: Missing link

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md (hunk 1 above)
Comment (GregSutcliffe) [reply _posts/2018-08-31-foreman-community-newsletter---august-2018.md]: Fixed, ty

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md (hunk 2 above)
Comment (GregSutcliffe) [reply _posts/2018-08-31-foreman-community-newsletter---august-2018.md]: Fixed, ty

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md (hunk 4 above)
Comment (GregSutcliffe) [reply _posts/2018-08-31-foreman-community-newsletter---august-2018.md]: Fixed, ty

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md (hunk 3 above)
Comment (GregSutcliffe) [reply _posts/2018-08-31-foreman-community-newsletter---august-2018.md:25]: OK, fair enough. Now I need to go do nursery pickup - if the release is going out today, feel free to add it in.

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+and then dive into details from there.
+
+I'd recommend for *anyone* with an interest in the future direction of Foreman
+to read over these, and give their thoughts on any part of it which interests
+you. Thanks!
+
+### Foreman at FrOSCon
+
+Thanks to the awesome work of Matthias, Bernhard, and the rest of the ATIX
+team, Foreman had a community presence at FrOSCon in Bonn this year. Whilst I
```
Comment (sbernhard) [inline _posts/2018-08-31-foreman-community-newsletter---august-2018.md:59]: ....presence at **FrOSCon at the University of Applied Sciences Bonn-Rhein-Sieg** this year.

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md (hunk 3 above)
Comment (ekohl) [reply _posts/2018-08-31-foreman-community-newsletter---august-2018.md:25]: Today is labor day in the US so I don't think so. You could mention it's in the process of being released.

**Comment by GregSutcliffe** [comment]
Merged, thanks all!

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md (hunk 5 above)
Comment (GregSutcliffe) [reply _posts/2018-08-31-foreman-community-newsletter---august-2018.md:59]: added, ty

File: _posts/2018-08-31-foreman-community-newsletter---august-2018.md (hunk 3 above)
Comment (GregSutcliffe) [reply _posts/2018-08-31-foreman-community-newsletter---august-2018.md:25]: Ok, I noted it in the release section, thanks

**Pull request closed without merging**